src/MainPage/Components/Navigation/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./style.scss";
```

Comment:
Isti komentar kao prethodni
```suggestion
import './style.scss';
```